explainer/Explainer11.md

```diff
@@ -0,0 +1,287 @@
+
+[![IMAGE ALT TEXT](http://img.youtube.com/vi/lt_P2BU8e3I/0.jpg)](http://www.youtube.com/watch?v=lt_P2BU8e3I "TD usage with a programming API")
+
+This example also shows that TDs can be used to easily onboard Things into any IoT ecosystems (e.g., IoT cloud services, edge systems, etc.). TDs helps to understand what the Things offers and how the offered data and functions can be bound to applications processes (e.g., creating service mash-ups, dashboards, etc.). The following example shows how TD are used in a node-red application:
+
+[![IMAGE ALT TEXT](http://img.youtube.com/vi/oAcYbJ6P9bU/0.jpg)](http://www.youtube.com/watch?v=oAcYbJ6P9bU "TD usage in node-red")
+
+TD also helps to mange your IoT system that typically consist of a heterogeneous device landscape based on different vendors and technologies that are used (e.g., HTTP, MQTT, Modbus, etc). Directories can be used to manage the TDs (e.g., create, search, etc). [The W3C WoT Discovery](https://w3c.github.io/wot-discovery/) provides details about a standardized how the interface of such directories can look like.  
+
+
+## What else is possible with the Thing Description specification?
+Besides the definitions of how a Thing description should be designed, there are other features provided around the TD, such as the Protocol Binding concept, TD context extensions, and Thing Model definitions. 
+
+### Protocol Binding
+Thing Description is not limited to http based interfaces as shown in the example above. In general, WoT is a protocol agnostic approach and provides a common mechanism to define how specific protocols such as MQTT, HTTP, CoAP or Modbus can be mapped to the WoT’s interaction properties-action-event abstraction within the Thing Description forms definition. More details is provide in the [Protocol Binding section](https://w3c.github.io/wot-thing-description/#protocol-bindings).  
+ 
+
+### TD Context Extensions
+Through JSON-LD serialization, the WoT Thing Description provides the ability to add context knowledge from additional namespaces. This mechanism can be used to enrich the Thing Description instances with additional (e.g., domain-specific) semantics. It can also be used to import additional Protocol Bindings or new security schemes in the future. 
+
+ The following TD extends the TD above by introducing a second definition in the @context to declare the prefix saref as referring to SAREF. This IoT ontology includes terms interpreted as semantic labels that can be set as values of the @type field, giving the semantics of Things and their interaction affordances. In the example below, the Thing is labelled with saref:LightSwitch, the status Property is labelled with saref:OnOffState and the toggle Action with saref:ToggleCommand.
```

Comment:
the prefix `saref`

Maybe also put `@context` in code-quotes, `saref:LightSwitch`, etc.

Reply:
done